vagrant/README.md

```diff
@@ -0,0 +1,15 @@
+Spotlight for Exhibits Demo on Vagrant
```

Comment:
I'm just generally curious what your thoughts are as to the role of this `README` as opposed to the project root version. Do we need both? Should this one hold the more vagrant specific things ( so more easily reused between projects for example ) and the root level be more Spotlight specific stuff? Just kinda thinking out loud.

Reply:
Right now it's part of spotlight. When we apply it to something else (Hyrax, looking at you) we can learn where the specifics are, and abstract them away, and make something more generic. Perhaps even as a sub-module (even though I don't really like sub-modules)

Reply:
Sounds good to me